src/test/java/com/thebois/testutils/MockUtils.java

```diff
@@ -0,0 +1,9 @@
+package com.thebois.testutils;
+
+public final class MockUtils {
+
+    private MockUtils() {
+
+    }
+
+}
```

Comment:
Huh?

Reply:
... I was supposed to move some things into here...


...


I never did.